docs/mpcnc/intro.md

```diff
@@ -0,0 +1,69 @@
+
+[!embed](https://www.youtube.com/watch?v=qJfYTv88YvI)
+
+This machine can easily do many things, how about making some coffee with Apach…(not me but I think he is my new hero).
+
+ 
+## Size
+
+- All axis can be any length you prefer, anything over 3 1/2′ (1M) would require small mid-span supports to increase rigidity, of course smaller is better.  The kit comes with enough belt for up to 48″ of total outer X and Y axis dimensions (eg 24″x24″, 36″x12″ or any other combination).
+
+- The smaller you make this the faster you can move it and the more rigid/accurate it will be, and more importantly the easier it will be to get the desired accuracy.
+
+- The linear motion is ball bearings on electrical conduit (or stainless steel)… seriously, easy to use and source. More information on this [here](https://www.v1engineering.com/assembly/machine-size/). If this machine isn’t big enough I have also designed the [LowRider CNC](https://www.v1engineering.com/lowrider-cnc/), all the same specs and resolution just designed to be bigger!
+
+ 
+## Power
+
+- X and Y axis are powered by 2 stepper motors each, and a single stepper for the Z axis. The standard would be NEMA 17 in any torque preferably 42 OZ/in and above (the kit come with 76 OZ/in). No need for NEMA 23’s or their required larger drivers,  torque is not one of this machines issues.
+
+- Belts are used for there accuracy and price. Small belts are used because each axis has two of them. Ballscrews are expensive, require tuning, and periodic maintenance / adjustments.
+
+ 
+## Control
+
+- This is all controlled by any control board you like. The UltiMachine boards are recommended for the improved design and robust safety features. You can still use the common Ramps 1.4 or any others boards as well. There are other [Marlin](https://github.com/MarlinFirmware/Marlin) based boards, [GRBL](https://github.com/grbl/grbl), and regular CNC boards as well.
+
+ 
+## Software
+
+- There are plenty of free or really inexpensive software options available for 3D printing, CAM, and CAD. I suggest [Fusion 360](http://www.autodesk.com/products/fusion-360/overview), [Repetier-host](http://www.repetier.com/), [ESTLCam](http://www.estlcam.com/), ect.
+
+ 
+## DIY
+
+- Besides using either common imperial or metric hardware, the rest of the machine is easily 3D printable, RepRap style! Full list of hardware [here](https://www.v1engineering.com/blog/parts/), and the files for the printed parts can be had [here](https://www.thingiverse.com/thing:724999).
```

Comment:
This points to the Burly. Are you going to move the Primo parts page to the docs?